src/components/Comments/WriteComment.js

```diff
@@ -222,7 +221,7 @@ export const styles = StyleSheet.create({
     flex: 1,
     fontSize: 12,
     fontFamily: fonts.inter[400],
-    color: colors.black,
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color in the `content` style has been updated. Ensure that this new color provides sufficient contrast for readability.
```diff
- color: colors.black,
+ color: COLORS.black,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->